pages/examples/tabs.py

```diff
@@ -0,0 +1,43 @@
+from dash import Dash, dcc, html
+from plotly import data
+import plotly.express as px
+
+# Tab 1
+gapminder = data.gapminder()
+fig_gapminder = px.scatter(
+    gapminder.query("year==2007"),
+    x="gdpPercap",
+    y="lifeExp",
+    size="pop",
+    color="continent",
+    hover_name="country",
+    log_x=True,
+    size_max=60)
+
+# Tab 2
+iris = data.iris()
+fig_iris = px.scatter(
+        iris,
+        x="sepal_length",
+        y="sepal_width", 
+        color="species")
+
+# App
+app = Dash(__name__)
```

Comment:
let's use dbc theme please, put everything inside container, and use dbc.Row and dbc.Col instead of html.Div